code/datums/gamemode/objectives/target/assassinate.dm

```diff
@@ -44,8 +53,47 @@
 		return TRUE
 	//Borgs/brains/AIs count as dead for traitor objectives. --NeoFite
 	// If they're in an away mission/custom z-level by the time you're checking this might as well count them as MIA
+	switch (syndicate_checked)
+		if (SYNDICATE_VALIDATED)
+			return TRUE
+		if (SYNDICATE_CANCELED)
+			return FALSE
 	if(target && target.current)
 		if(target.current.stat == DEAD || issilicon(target.current) || isbrain(target.current) || target.current.z > 6 || !target.current.ckey || isborer(target.current))
 			return TRUE
 		return FALSE
-	return TRUE
\ No newline at end of file
+	return TRUE
+
+/datum/objective/target/assassinate/proc/SyndicateCertification()
+	if (syndicate_checked == SYNDICATE_CANCELED)//if your death was reported prior, you're out of the race. Better luck next time.
+		return
+
+	syndicate_checked = SYNDICATE_VALIDATED
+
+	//The syndicate has confirmed that the double agent has taken out their target.
+	//They will now assign the new objective of assassinating their old target's target.
+	//Unless said target is themselves, which then means that all other agents have been eliminated and they have won.
+	var/datum/role/traitor/challenger/self = owner.GetRole(CHALLENGER)
+	var/datum/role/traitor/challenger/enemy = target.GetRole(CHALLENGER)
+	if (!self ||!enemy)
+		message_admins("a double agent syndicate certification has failed. call deity.")//not that this should ever happen
```

Comment:
Don't be a Carn, remove your name here

Reply:
I don't really see how it's an issue since even when I take a break from space I'm always available to fix bugs. Wouldn't you want the person who has to fix a bug to preferably be the one who wrote the piece of code when they're available?

Reply:
The only complaint I have with this is that the likelyhood of anybody being able to figure out what happened in a situation like this is extremely unlikely. I feel like this would most likely just crop up from like... adminbus maybe??

Reply:
Pretty much, I'll remove the message altogether.